packages/trader/src/sass/app/_common/mobile-widget.scss

```diff
@@ -23,6 +24,14 @@
         @include typeface(--paragraph-center-normal-black);
         color: var(--text-less-prominent);
     }
+    &__wrapper {
+        display: flex;
+
+        div:last-child:not(:only-child) {
```

Comment:
Same as above. Needs to use the `weak` property in class defined on [L3](https://github.com/binary-com/deriv-app/pull/838/files#diff-160bce3b3f4ce2a13fcaaa5a219a592fR3).